easybuild/easyconfigs/g/glog/glog-0.7.1-GCCcore-13.3.0.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'CMakeMake'
+
+name = 'glog'
+version = '0.7.1'
+
+homepage = 'https://github.com/google/glog'
+description = "A C++ implementation of the Google logging module."
+
+toolchain = {'name': 'GCCcore', 'version': '13.3.0'}
+toolchainopts = {'cstd': 'c++14'}
+
+source_urls = ['https://github.com/google/glog/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['00e4a87e87b7e7612f519a41e491f16623b12423620006f59f5688bfd8d13b08']
+
+builddependencies = [
+    ('binutils', '2.42'),
+    ('CMake', '3.29.3'),
+]
+
+dependencies = [
+    ('gflags', '2.2.2'),
+    ('libunwind', '1.8.1'),
+]
+
+configopts = '-DBUILD_SHARED_LIBS=ON '
```

Comment:
Use build_shared_libs = True instead

Reply:
In PR [24853](https://github.com/easybuilders/easybuild-easyconfigs/pull/24853) which got merged it was set to `ON` so for consistency I would suggest to leave it like that.

Reply:
Also think since a version in a newer TC uses this even if it is not the suggest way does not make sense to fix this in an older TC